t/access_log.t

```diff
@@ -46,7 +50,8 @@ my %common = (
 );
 
 # Should be ignored, behind_proxy defaults to disabled.
-$jmap_tester->ua->default_header('X-Forwarded-For' => '1.2.3.4');
+# XXX This is assuming an LWP UA. -- rjbs, 2019-04-24
+$jmap_tester->ua->lwp->default_header('X-Forwarded-For' => '1.2.3.4');
```

Comment:
Yeah... we will probably want to proxy some of the methods we expect folks to use on the ua - light mucking with cookies and headers...

Reply:
This should now use `set_default_header` and not call `->lwp`.

Reply:
I did the heck out of both those requests.